lua/ogpt/flows/actions/base.lua

```diff
@@ -147,16 +147,16 @@ function BaseAction:update_variables()
   end
 end
 
-function BaseAction:render_template(variables, templates)
+function BaseAction:render_template(variables)
   variables = vim.tbl_extend("force", self.variables, variables or {})
   -- lazily render the final string.
   -- it recursively loop on the template string until it does not find anymore
   -- {{{}}} patterns
   local stop = false
   local depth = 2
-  local result = self.template
+  local template = self.template
```

Comment:
`result` sounds ambiguous. `template` must do the job.

Reply:
definitely, there are lots of clean up to do. the 'dev' branch have a lot of updates that i'm gonna roll out. so i'll merge these changes in for now. but note the Dev branch is where things are at.